### ISSUE TITLE
outputDir without allure doesn't seem to work

### ISSUE THREAD

**Issue by Waltari10** [presidenten/wdio-video-reporter#49]
**Describe the bug**
Everything else works like a charm, except the outputDir setting doesn't seem to do much. It creates a "videos" folder at the given path, but everything is still saved under "./_results_".

**To Reproduce**
```
  reporters: [
    [video, {
      outputDir: './e2e/test_output/videos', // Doesn't save under this location for some reason...
      saveAllVideos: false,       // If true, also saves videos for successful test cases
      videoSlowdownMultiplier: 3, // Higher to get slower videos, lower for faster videos [Value 1-100]
    }],
    'spec',
    [
      'cucumberjs-json',
      {
        jsonFolder: './e2e/test_output',
      },
    ],
  ],
```

**Expected behavior**
I would except all the videos and screenshots to be saved under './e2e/test_output/videos'.

**Environment (please complete the following information):**
 - **wdio-video-reporter version:** ^3.1.0
 - **WebdriverIO version:** ^5.23.0
 - **Node.js version:** v12.18.3
 - **NPM version:** 6.14.8 - **Platform name and version:** MacOS Catalina Version 10.15.6

**Smartphone (please complete the following information):**
 - Device: iPhone 11 simulator
 - OS: 13.0

** Additional info **
We are using Appium and Cucumber with the E2E tests.

**Comment by presidenten** [comment]
This one sure got me this morning :-)
I need more coffee.

At one point I decided to not have a separate outputDir property in the reporter and just use the outputDir from wdio. ...but it seems like I forgot to update the readme. :-)

However, it is easy to get it back into the code if there is a need to have a separate outputDir for this reporter.

**Comment by presidenten** [comment]
Just figured you might not care about wdio logs, but still want to direct the videos.
So I fixed it to be according to the readme again.

Issue is resolved in v3.1.1 that is now availble on npm
